input_parameters.conf

```diff
@@ -103,6 +103,13 @@ noDataValue:   0.0
 # Nan conversion flag. Set to 1 if missing No-data values are to be converted to NaN
 nan_conversion: 1
 
+
+# sign convention for phase data
```

Comment:
Correct comments - subsidence as negative and uplift as positive mean the same thing.

Also maybe add a disclaimer that the sign convention only works as stated if inputs are in standard GAMMA format?

Reply:
N.B. there is no "standard" GAMMA format ;-)

Reply:
Well in that case maybe we want something like this?
```
+1 - keep sign convention of input interferograms
-1 - reverse sign convention compared to input interferograms
```
We don't want a user putting in interferograms where negative values already correspond to motion away from the satellite, and expecting `signal_polarity: -1` to preserve that.

Reply:
Yes great point. We still need our user to understand what their input data signal convention is.

Reply:
I have updated `input_parameters.conf` based on @richardt94's idea